_docs/performance-tuning/drill-metastore/010-using-drill-metastore.md

```diff
@@ -117,6 +119,13 @@ Drill can connect to any number of data sources, each of which may have its own
 As a result, the Metastore labels tables with a combination of (plugin configuration name, workspace name, table name).
 Note that if before renaming any of these items, you must delete table's Metadata entry and recreate it after renaming.
 
+### Using schema provisioning feature with Drill Metastore
+
+Drill Metastore cooperates with [Schema provisioning]({{site.baseurl}}/docs/create-or-replace-schema/#usage-notes) feature.
```

Comment:
I think instead of cooperates, it better to expand: Drill Metastore allows to specify using the same syntax as Schema provisioning feature when used as table function (add link to the docs).